### PR TITLE
Created my Profile Ansh-Rathod

### PR DESCRIPTION
**Pull request by Ansh-Rathod**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2297"></a>

### CODE REVIEW

File: data/Ansh-Rathod.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AnshhRathod",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/ansh-rathod",
+      "icon": "linkedin"
+    }
+
```
Comment (schmelto) [inline data/Ansh-Rathod.json:33]:
```suggestion
```

File: data/Ansh-Rathod.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Ansh Rathod",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full-Stack developer | Flutter | Nextjs",
+  "avatar": "https://github.com/Ansh-Rathod.png",
+  "tags": ["flutter","nodejs","dart","javascript", "typescript"],
```
Comment (schmelto) [inline data/Ansh-Rathod.json:7]:
```suggestion
  "tags": ["flutter", "nodejs", "dart", "javascript", "typescript"],
```

File: data/Ansh-Rathod.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    {
+      "title": "Started Developing and open-source",
+      "date": "December 2020",
+      "icon": "github",
+      "color": "black",
+      "description": "One of my best and life changing decision ever.",
+      "url": "https://github.com/Ansh-Rathod"
+    }
+  ]
+}
```
Comment (Pradumnasaraf) [inline data/Ansh-Rathod.json:60]:
```suggestion
}

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, an ending line is required, otherwise looks to me

Comments: Fix the failing action

**Comment by Ansh-Rathod** [comment]
> Hey, an ending line is required, otherwise looks to me
> 
> Comments: Fix the failing action

cool, thanks!

**Review by kumarsonsoff3 — APPROVED** [review]
Amazing collaboration! 🎉
Looks Good to me! 👍

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by stevenovak123 — APPROVED** [review]

**Review by stevenovak123 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**